### DISCUSSION TITLE
How to change number of columns?

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#5943] [Q&A] [answered]
When I increase the base font size (with something like `--ifm-font-size-base: 120%;`) the table of contents on the right is no longer wide enough. Messing around in the browser inspector, I see that by default (on a blog page):
- the main content (a `<main>`) is 7 columns wide because it has class `col--7`, and
- the table of contents (a `<div>`) is 2 columns wide because it has class `col--2`.
If I change in inspector the main content to be 6 columns and the table of contents to be 3 columns wide, my problem is solved. But I don't know how to do this in a way that isn't hopelessly hacky and brittle.

Does anyone know a way?

**Comment by Josh-Cena** [comment, marked as answer]
The only way to do this for now is to swizzle `BlogLayout` and change the class names. You shouldn't be deterred by the unsafe warnings—they are only meant to be reminders, not blockers

**Comment by ghost** [comment]
Thanks @Josh-Cena. I was deterred at first by the warnings, so it's good to hear I need not be.
